gapic-generator-cloud/templates/cloud/service/client/_credentials.erb

```diff
@@ -6,12 +6,10 @@ require "googleauth"
 class <%= service.credentials_name %> < Google::Auth::Credentials
   <%- if service.client_scopes.any? -%>
   self.scope = [
-    <%- client_scopes = service.client_scopes.dup -%>
-    <%- last_client_scope = client_scopes.pop -%>
-    <%- client_scopes.each do |client_scope| -%>
-    <%= client_scope.inspect %>,
-    <%- end -%>
-    <%= last_client_scope.inspect %>
+  <%- service.client_scopes.each_with_index do |client_scope, index| -%>
```

Comment:
Here is the new implementation for correctly appending the comma.

Reply:
Same thing, can we use a trinary operator or otherwise ensure that `comma` gets actively set on the final iteration?